frameworks/digital-outcomes-and-specialists/questions/briefs/technicalCompetenceCriteriaOutcomes.yml

```diff
@@ -9,11 +9,14 @@ question_advice: |
   - nice-to-have skills and experience
   - proposal
 
-  ##Scoring (optional)
+  ##Giving points for individual criteria (optional)
 
-  If you want to score individual criteria, you must add points for everything. Read [how to score individual criteria](https://www.gov.uk/guidance/how-to-set-your-evaluation-criteria-when-buying-digital-outcomes-and-specialists-services#giving-points-to-individual-criteria).
-
-  The points you give for nice-to-have skills and experience must be lower than the points you give to the supplier’s essential skills and experience and proposal.
+  You can give points to individual criteria, eg ‘have experience designing services for users with low digital
+  literacy – 15 points’. If you choose to add points, you must do it for all individual criteria.
+  
+  The points you give for nice-to-have skills and experience criteria must be lower than the points you give for 
+  essential skills and experience and proposal criteria.
+  Read [how to score individual criteria](https://www.gov.uk/guidance/how-to-set-your-evaluation-criteria-when-buying-digital-outcomes-and-specialists-services#giving-points-to-individual-criteria).
```

Comment:
<sub>Tiny point</sub>

Do we care about "[Read how](https://github.com/alphagov/digitalmarketplace-frameworks/search?utf8=%E2%9C%93&q=%22read+how%22&type=Code)" versus "[Read about how](https://github.com/alphagov/digitalmarketplace-frameworks/search?utf8=%E2%9C%93&q=%22read+about+how%22&type=Code)"?